pyccl/core.py

```diff
@@ -164,13 +167,15 @@ class Cosmology(object):
             Defaults to 'tinker10' (2010).
         halo_concentration (:obj:`str`, optional): The halo concentration
             relation to use. Defaults to Duffy et al. (2008) 'duffy2008'.
-        emulator_neutrinos: `str`, optional): If using the emulator for
+        emulator_neutrinos (:obj: `str`, optional): If using the emulator for
             the power spectrum, specified treatment of unequal neutrinos.
             Options are 'strict', which will raise an error and quit if the
             user fails to pass either a set of three equal masses or a sum with
             m_nu_type = 'equal', and 'equalize', which will redistribute
             masses to be equal right before calling the emulator but results in
             internal inconsistencies. Defaults to 'strict'.
+        extra_parameters (:obj:`dict`, optional): Dictionary holding extra
+            parameters. Defaults to None.
```

Comment:
Maybe explain that these will be understood by CAMB (and we may update to also CLASS later on)

Reply:
What markup format is being used for the docstrings? I'd to list the currently supported parameters but to so with sane formatting.

Reply:
I'd say that's what the CAMB documentation is there for. Here I'd just say "these can currently include any parameters that can be parsed by CAMB, but will be extended in the future" or something like that.